src/bundle/ui-dev/src/modules/multi-file-upload/components/upload-list/upload.item.component.js

```diff
@@ -518,22 +372,19 @@ export default class UploadItemComponent extends Component {
         this.contentInfoInput.value = contentId;
         this.contentVersionInfoInput.value = contentId;
         this.contentVersionNoInput.value = versionNo;
+
         window.document.querySelector(`#form_subitems_content_edit_language_${languageCode}`).checked = true;
+
         this.contentEditBtn.click();
     }
 
-    /**
-     * Renders an edit content button
-     *
-     * @method renderEditBtn
-     * @memberof UploadItemComponent
-     * @returns {null|Element}
-     */
     renderEditBtn() {
-        const { uploaded, aborted, disallowedSize, disallowedType, failed, uploading } = this.state;
-        const canEdit = this.props.isUploaded || (uploaded && !aborted && !(disallowedSize || disallowedType) && !failed && !uploading);
+        const Translator = getTranslator();
+        const { instanceUrl } = getRestInfo();
+        const { uploaded, aborted, failed, uploading } = this.state;
+        const canEdit = (this.props.isUploaded || (uploaded && !aborted && !uploading)) && !failed;
```

Comment:
Sth is not ok in logic here for me - how can file be both upladed and uploading? If it cant, then there's no point in checking both uploaded and !uploading